docs/book/component-guide/service-connectors/connector-types/docker-service-connector.md

```diff
@@ -91,9 +91,5 @@ The 'dockerhub' Docker Service Connector connector was used to successfully conf
 
 The Docker Service Connector can be used by all Container Registry stack component flavors to authenticate to a remote Docker/OCI container registry. This allows container images to be built and published to private container registries without the need to configure explicit Docker credentials in the target environment or the Stack Component.
 
-{% hint style="warning" %}
```

Comment:
This warning is still valid for every orchestrator apart from the Kubernetes one for which you now implemented it. So either this gets implemented for all orchestrators as part of this PR, or this warning needs to stay.